lib/open_food_network/users_and_enterprises_report.rb

```diff
@@ -11,13 +11,13 @@ def initialize(params = {})
 
     def header
       [
-        I18n.t(:report_header_user),
-        I18n.t(:report_header_relationship),
-        I18n.t(:report_header_enterprise),
-        I18n.t(:report_header_is_producer),
-        I18n.t(:report_header_sells),
-        I18n.t(:report_header_visible),
-        I18n.t(:report_header_confirmation_date),
+        "User",
```

Comment:
These changes are unrelated as well, right?

Reply:
The rest of the changes in this file are related. Comparing this file with other branches it looks like the translated versions of the headers were added more recently than this branch was created. I'll add a commit now to make sure the translated versions are included.

Reply:
Just for clarity, the changes to this file were needed because `enterprise.confirmed_at` no longer exists as a field.